components/ws-manager-bridge/src/cluster-service-server.ts

```diff
@@ -191,31 +188,8 @@ export class ClusterService implements IClusterServiceServer {
                                         if (v.level === (c as AdmissionConstraintHasUserLevel).level) {
                                             return false;
                                         }
-                                }
-                                return true;
-                            })
-                        }
-                    }
-                }
-                if (call.request.hasAdmissionPreference()) {
```

Comment:
:+1: Really like this going away. Having just `admissionContraints` is IMO easier to wrap yur head around.